wolfcrypt/src/port/st/stsafe.c

```diff
@@ -170,7 +170,7 @@ int SSL_STSAFE_VerifyPeerCertCb(WOLFSSL* ssl,
         #ifdef USE_STSAFE_VERBOSE
             STSAFE_INTERFACE_PRINTF("stsafe_interface_verify error: %d\n", err);
         #endif
-            err = WC_HW_E;
+            err = -err;
```

Comment:
There is another place this logic is done for crypto callback. Consider fixing there too

Reply:
I found similar logic at line 507 and will make the same adjustment there.